subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_lte_notification.c

```diff
@@ -92,3 +101,5 @@ void lte_notify_handler(const struct lte_lc_evt *const evt)
 		break;
 	}
 }
+
+SYS_INIT(lwm2m_lte_lc_handler_register, APPLICATION, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
```

Comment:
The initialization level is `APPLICATION`, but the priority uses the kernel default value. Maybe `CONFIG_APPLICATION_INIT_PRIORITY` would be more appropriate? Probably doesn't affect the functionality, though.